src/app/components/collectibles/collectible-image.tsx

```diff
@@ -13,36 +15,34 @@ export function CollectibleImage(props: CollectibleImageProps) {
   const [isError, setIsError] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
   const [width, setWidth] = useState(0);
-
-  if (isError)
-    return (
-      <CollectibleItemLayout collectibleTypeIcon={icon} {...rest}>
-        <ImageUnavailable />
-      </CollectibleItemLayout>
-    );
+  const isImageAvailable = src && isValidUrl(src);
```

Comment:
If the issue is that src is undefined, we have a mismatch between runtime and our types

Reply:
It comes back as `undefined` for video / audio NFT's only. Maybe I need to add a new type for those NFTs? 

We can't play them anyway as the `token_uri` is not giving us a valid video file. 

<img width="664" alt="Screenshot 2025-01-27 at 15 56 45" src="https://github.com/user-attachments/assets/9df32e71-dca1-404c-be18-654d1977ef87" />

This change basically keeps the code as it was before but displays the metadata we do have rather than a larger `Image not found` card:
<img width="466" alt="Screenshot 2025-01-29 at 10 46 58" src="https://github.com/user-attachments/assets/549db30d-790d-49b4-8bc4-655c7b1ef918" />

Reply:
zod schema checking here would be great too, but so long as the types align with the runtime I'm happy